### ISSUE TITLE
Doesn't cross compile

### ISSUE THREAD

**Issue by fireboy1919** [simplereach/timeutils#3]
I got this error when attempting to build this with goxc:
[goxc:xc] 2016/04/21 11:33:30 Stopping after 'xc' failed with error 'Wait error: exit status 1: ../zeppelin/paragraph.go:5:2: C source files not allowed when not using cgo or SWIG: approxidate.c'
[goxc] 2016/04/21 11:33:30 RunTasks error: Wait error: exit status 1: ../zeppelin/paragraph.go:5:2: C source files not allowed when not using cgo or SWIG: approxidate.c

Perhaps there's no way around this, but it does mean that one of the most convenient ways of making portable go code is not available when used with this library.

**Comment by elubow** [comment]
That's correct.  The problem is that cross compiling in general isn't really usable with CGo.  It isn't something specific to this library.
